lib/node_modules/@stdlib/stats/base/variancewd/lib/index.js

```diff
@@ -45,11 +45,16 @@
 
 // MODULES //
 
-var main = require( './main.js' );
+var setReadOnly = require( '@stdlib/utils/define-nonenumerable-read-only-property' );
+var variancewd = require( './variancewd.js' );
+var ndarray = require( './ndarray.js' );
 
 
-// EXPORTS //
+// MAIN //
+
+setReadOnly( variancewd, 'ndarray', ndarray );
 
-module.exports = main;
 
-// exports: { "ndarray": "main.ndarray" }
```

Comment:
You need to keep these exports comment. It is intentional and should follow L60.